README.md

```diff
@@ -35,19 +43,21 @@ As can a particular commit hash:
 
 ### Installing a DMS release ###
 
-We still package our releases to DMS via a conda spec file that lists the exact versions of all packages to be installed.
-To create a new environment with a specific release, use:
-
-    conda create -n jwst_env --file <URL>
-    conda activate jwst_env
+We still package our releases to DMS via environment snapshots that specify the
+exact versions of all packages to be installed.  Each such delivery has its own
+installation instructions which may be found in the corresponding release
+documentation linked from this page:
 
-where `<URL>` is of the form:
+    https://github.com/astroconda/astroconda-releases/tree/master/jwstdp
 
-    Linux: http://ssb.stsci.edu/releases/jwstdp/0.13.7/latest-linux
-    OS X: http://ssb.stsci.edu/releases/jwstdp/0.13.7/latest-osx
+The version values shown there are the JWSTDP releases available to install.
+Installation procedures for each version are in the README.md file for that
+version. See the "Software vs DMS build version map" table below for a
+list of version tags corresponding to particular releases.
```

Comment:
I like the additional text that explains some more detail about the DMS release.  

I think though it is preferable to keep this as the one place for installation instructions, and this should be kept up to date.  Please add the instructions for the current latest release here as well.